backend/globaleaks/utils/backup.py

```diff
@@ -0,0 +1,89 @@
+
+import json
+import os
+
+from globaleaks.utils.utility import iso8601_to_datetime
+from globaleaks import models
+
+
+def get_backups_parameter(session):
+    backup_enable_config = session.query(models.Config.value).filter(
+        models.Config.tid == 1, models.Config.var_name == 'backup_enable').one_or_none()
+    backup_time_iso_8601_config = session.query(models.Config.value).filter(
+        models.Config.tid == 1, models.Config.var_name == 'backup_time_ISO_8601').one_or_none()
+    backup_destination_path_config = session.query(models.Config.value).filter(
+        models.Config.tid == 1, models.Config.var_name == 'backup_destination_path').one_or_none()
+
+    backup_enable = backup_enable_config.value if backup_enable_config else None
+    backup_time_iso_8601 = backup_time_iso_8601_config.value if backup_time_iso_8601_config else None
+    backup_destination_path = backup_destination_path_config.value if backup_destination_path_config else None
+
+    return (backup_enable, backup_time_iso_8601, backup_destination_path)
+
+
+def get_audit_log_file_path(backup_destination_path):
+    return os.path.join(backup_destination_path, 'audit.log')
+
+
+def write_audit_log_on_file(session, row):
```

Comment:
This implementation is very interesting.
We will probably integrate it it saving the audit log always in /var/globaleaks/auditlog.log even if the backup is disabled making the implementation independent from the backup.
This way the backup functionality will naturally create a copy of the file on the backup server while running the backups.